docs/serving/using-custom-metrics-API.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Using the custom-metrics API"
+weight: 10
+type: "docs"
+---
+
+The `custom-metrics` API is installed by default when you install Knative Serving.
```

Comment:
What does the custom metrics API allow users to do?

Reply:
It allows users to make use of concurrency based scaling in the **HPA**.

Reply:
@abrennan89  Could you add this info to this page?